.ci/validation.sh

```diff
@@ -234,6 +234,24 @@ no-error-pmd)
   removeFolderWithProtectedFiles pmd
   ;;
 
+no-error-hazelcast)
+  CS_POM_VERSION="$(getCheckstylePomVersion)"
+  echo "CS_version: ${CS_POM_VERSION}"
+  ./mvnw -e --no-transfer-progress clean install -Pno-validations
+  echo "Checkout Hazelcast sources..."
+  checkout_from "https://github.com/hazelcast/hazelcast.git"
+  cd .ci-temp/hazelcast
+  # Get absolute path to checkstyle config
+  CHECKSTYLE_ROOT_DIR="$(cd ../../.. && pwd)"
+  CONFIG_PATH="${CHECKSTYLE_ROOT_DIR}/config/checkstyle-checks.xml"
+  echo "Using config at: ${CONFIG_PATH}"
+  mvn -e --no-transfer-progress checkstyle:check \
+    -Dcheckstyle.version="${CS_POM_VERSION}" \
+    -Dcheckstyle.config.location="${CONFIG_PATH}"
```

Comment:
We should use config from hazelcast repo.
We just need to override version, nothing else.

Reply:
It is strange that mvn execution is passing 
Please revert commit https://github.com/hazelcast/hazelcast/commit/c4ed533167f462918ab19ebc2dbdcfeb7bcc1d0b#r157717810
Before run, it should fail , as suppression should be reverted, so there should be violation and failed ci